app/src/main/res/values-fr/strings.xml

```diff
@@ -340,30 +340,30 @@
-    <string name="node_name_hint">Node Name (Optional)</string>
-    <string name="node_address_hint">Hostname</string>
+    <string name="info_nodes_enabled">Nœud-tomatiC activé, tapez pour plus d\'infos.</string>
+    <string name="node_height">Dernier bloc mis à jour : %1$s</string>
+    <string name="label_nodes">Nœuds</string>
+    <string name="node_name_hint">Nom du Nœud (Optionnel)</string>
+    <string name="node_address_hint">Nom d\'hôte</string>
     <string name="node_port_hint">Port</string>
-    <string name="node_user_hint">Username (Optional)</string>
-    <string name="node_pass_hint">Password (Optional)</string>
-    <string name="node_host_unresolved">Cannot resolve host</string>
-    <string name="node_host_empty">We need this!</string>
-    <string name="node_port_numeric">Must be numeric</string>
-    <string name="node_port_range">Must be 1&#8211;65535</string>
-    <string name="node_fab_add">Add Node</string>
-    <string name="node_refresh_hint">Touch to refresh!</string>
-    <string name="node_test_error">CONNECTION ERROR %1$d</string>
-    <string name="node_general_error">CONNECTION ERROR</string>
-    <string name="node_auth_error">AUTHENTICATION FAILED</string>
-    <string name="node_result_label">Test Result:</string>
-    <string name="node_result">Height: %1$s (v%2$d), Ping: %3$.0fms, IP: %4$s</string>
-    <string name="node_testing">Testing IP: %1$s &#8230;</string>
-    <string name="node_refresh_wait">Please wait for scan to finish</string>
-    <string name="node_create_hint">Touch to select or add nodes</string>
-    <string name="node_pull_hint">Add nodes manually or pull down to scan</string>
-    <string name="node_scanning">Scanning network&#8230;</string>
-    <string name="node_nobookmark">Automatically bookmarked best %1$d nodes</string>
-    <string name="label_test">Test</string><!--note: as in "Test a network connection"-->
+    <string name="node_user_hint">Nom d\'utilisateur (Optionnel)</string>
+    <string name="node_pass_hint">Mot de passe (Optionnel)</string>
+    <string name="node_host_unresolved">L\'hôte ne peut pas être résolu</string>
+    <string name="node_host_empty">On a besoin de ça !</string>
+    <string name="node_port_numeric">Doit être un nombre</string>
+    <string name="node_port_range">Doit être de 1 à 65535</string>
+    <string name="node_fab_add">Ajouter un Nœud</string>
+    <string name="node_refresh_hint">Toucher pour raffraichir !</string>
+    <string name="node_test_error">ERREUR DE CONNEXION %1$d</string>
+    <string name="node_general_error">ERREUR DE CONNEXION</string>
+    <string name="node_auth_error">ÉCHEC D\'AUTHENTIFICATION</string>
+    <string name="node_result_label">Résultat du test :</string>
+    <string name="node_result">Hauteur : %1$s (v%2$d), Ping : %3$.0fms, IP : %4$s</string>
+    <string name="node_testing">Test de l\'IP : %1$s &#8230;</string>
+    <string name="node_refresh_wait">Pateintez jusq\'à la fin du scan</string>
```

Comment:
Pateintez -> Patientez   
scan -> recherche ?

Reply:
ok for both